releng/docker/build-init.sh

```diff
@@ -109,6 +109,15 @@ if [ "${#PACKAGES[@]}" -gt 0 ]; then
   xbps-install -Sy "${PACKAGES[@]}"
 fi
 
+if [ -d /alt ]; then
```

Comment:
When bind mounting NFS shares into containers, some initramfs generators complain that they can't copy files because the file source (NFS) has additional permissions that can not be represented in the tmpfs based filesystems within the container. The workaround is to create an alterative root, and anything that is mounted here will be copied to /.

Reply:
can we leave this PR at the original scope?

Reply:
It's not clear what should be mounted at `/alt`, it isn't documented anywhere but this comment, and is outside the scope of this PR.